### ISSUE TITLE
How to process audio that exceeds 10 seconds in length

### ISSUE THREAD

**Issue by qisawO3** [YuanGongND/ltu#12]
Hello, I would like to ask, how do you test the audio in the LibriSpeech dataset that exceeds 10 seconds in duration?I'm encountering an issue while using the LibriSpeech dataset for speech recognition. Despite modifying the pad_or_trim method in the transcribe_audio function, I still face a problem with mismatched model dimensions. How can I resolve this issue?

**Comment by qisawO3** [comment]
Thank you for your response. I would like to ask further, for audio that exceeds 10 seconds in length, if only the first 10 seconds of audio features are used for ASR, then when calculating the WER during testing, should the label content also be truncated to the first 10 seconds?

**Comment by qisawO3** [comment]
So, let me give an example. My question is: What this audio segment is saying?does LTU-AS output the entire text, and not just the text contained within those 10 seconds of speech? In other words, is the ASR performance of LTU-AS entirely provided by **Whisper**?

**Comment by YuanGongND** [comment]
> What this audio segment is saying? does LTU-AS output the entire text, and not just the text contained within those 10 seconds of speech? 

No, LTU-AS almost never cut and output the spoken text in the first 10-second. In fact, it does not know what speech is in the first 10 seconds, as its acoustic input is Whisper **encoder** output, without the decoder, it cannot do ASR. 

> In other words, is the ASR performance of LTU-AS entirely provided by Whisper?

While it copies the Whisper output in most cases, it occasionally not follow instruction or add something not in the spoken text. Which makes its WER actually higher than Whisper output. Please check our paper in detail.

-Yuan
